### ISSUE TITLE
📣 [Promote]: Create blog post about different approaches that may be used for SPFx Development - compare using terminal vs SPFx Toolkit VS Code vs SPFx Toolkit GitHub Chat Participant

### ISSUE THREAD

**Issue by Adam-it** [pnp/vscode-viva#367]
### 🎯 Aim 

The aim is to create a blog post in which we may compare different tooling and approaches we may take to help us out in SPFx Development. A good example could be project scaffolding 
- we may use the standard `yo @microsoft/sharepoint` command to create a new project
- we may use SPFx Toolkit scaffolding form or sample gallery
- we may use SPFx Toolkit GitHub Chat Participant `/new` command

and setting up local workspace
- we may use CLI for Microsoft 365 [m365 spfx doctor](https://pnp.github.io/cli-microsoft365/cmd/spfx/spfx-doctor) command
- we may use SPFx Toolkit action
- we may use SPFx Toolkit GitHub Chat Participant `/setup command

we may also compare SPO management 
- using CLI for Microsoft 365
- in SPFx Toolkit that allows to manage apps in app cataloga
- using SPFx Toolkit GitHub Chat Participant `/manage` chat command 

Ideally, the blog post should be added to our community PnP blog: https://pnp.github.io/blog/

### 🙏 Help

Are you a regular user of SPFx Toolkit and you wanna help out but don't have any skills in developing VS Code extensions then this is a perfect issue for you! Help us out to promote the tool so that we may spread the word and reach more SPFx developers and help them rock 🤩

### Tip

Previous marketing things done are all grouped in this [wiki page](https://github.com/pnp/vscode-viva/wiki/2.-Marketing-resources). Adding them here for inspiration

**Comment by Adam-it** [comment]
> I’ll take this @Adam-it

All yours. Let me know if you need any suggestions about the chat participant. 
I suspect for the `/manage` chat command which is still beta there will be many 'glitches' so awesome to gather this feedback and open up an issue for improvements  😍. As for the `/setup` and `/new` chat commands you may find some examples in this release blog post https://pnp.github.io/blog/post/spfx-toolkit-vscode-v-4-2-release/

**Comment by Adam-it** [comment]
@Sandeep-FED I was wondering if you could provide some update in this issue? Do you need some help from my side to move this forward?

**Comment by Sandeep-FED** [comment]
Hi @Adam-it, Sorry for the delay. This is WIP. I will have the content ready in a couple of days. Please help me review it when it’s ready.

**Comment by Adam-it** [comment]
> Hi [@Adam-it](https://github.com/Adam-it), Sorry for the delay. This is WIP. I will have the content ready in a couple of days. Please help me review it when it’s ready.

sure thing 👍. Thanks a ton for the update

**Comment by Adam-it** [comment]
@Sandeep-FED could you provide what is the update on this one? Do you have some draft blog post we could check? Do you need some help to get you unblocked?
Or maybe something changed and if you may not commit to this one we may open this up for others 👍

**Comment by Adam-it** [comment]
@Sandeep-FED any update? Please 🙏

**Comment by Adam-it** [comment]
@Sandeep-FED sorry but due to lack of response and a stale sate of this issue I will open it up for other to be taken.
If anything would change and you would like to take action on this again, let me know 👍

**Comment by Adam-it** [comment]
I'll take it

**Comment by Sandeep-FED** [comment]
@Adam-it, have you started working on it? If not, please reassign this one to me, since the other blog was merged today.

**Comment by Sandeep-FED** [comment]
Lets go!!!

**Comment by Sandeep-FED** [comment]
@Adam-it I'm halfway done with the content, prolly take few more days as Im getting only few hours in a day to work on this.

**Comment by Adam-it** [comment]
> @Adam-it I'm halfway done with the content, prolly take few more days as Im getting only few hours in a day to work on this.

Thanks for the update👍
No pressure 👍

**Comment by Adam-it** [comment]
@Sandeep-FED did you manage to get any progress on this blog post?

**Comment by Sandeep-FED** [comment]
Yeah Adam, it is ready for PR will drop in couple of days.

**Comment by Sandeep-FED** [comment]
@Adam-it, Apologies for the delay! Will drop the PR tonight or tomorrow.

**Comment by Adam-it** [comment]
> @Adam-it, Apologies for the delay! Will drop the PR tonight or tomorrow.

No problem

**Comment by Sandeep-FED** [comment]
@Adam-it, please take your time on reviewing the content & let me know if any changes are required.

**Comment by Adam-it** [comment]
Thank you for the update. I will for sure try to give it a check in the coming days 👍

**Comment by Adam-it** [comment]
this was already done 
https://pnp.github.io/blog/post/different-approaches-in-spfx-development/
Forgot to close this one 😅
Once again, awesome work @Sandeep-FED on this 👏👏
